examples/biosnoop.c

```diff
@@ -0,0 +1,107 @@
+#include <uapi/linux/ptrace.h>
```

Comment:
If this code is taken from, or even just inspired by, code found in the BCC repo, we should cite it here. Example: https://github.com/rust-bpf/rust-bcc/blob/master/examples/softirqs.c#L3

Reply:
Oh I am quite sorry about this, I'll fix this and pay attention later.